### DISCUSSION TITLE
How do I delete a comment I don't want to send in?

### DISCUSSION THREAD

**Discussion by ag91** [wandersoncferreira/code-review#84] [Q&A] [answered]
I see my comment in the "Your Review Feedback" section of the Code Review buffer, but I cannot find a way to delete it.
I would expect to press "k" on it and it to disappear (like for Magit hunks): maybe there is already a way?

**Comment by wandersoncferreira** [comment, marked as answer]
@ag91 you can press RET on the feedback section and delete it. For other comments there is a command bound to `C-c C-k` that removes it. The idea to also remaps it to `k` is very good.

The feedback comment was supposed to be not empty always due to my bad interpretation of Github APIs for sending a review. There are some notes about this here https://github.com/wandersoncferreira/code-review/issues/59

I can also bound `k` to delete the feedback section now.

**Reply by wandersoncferreira** [reply]
@ag91 merged here https://github.com/wandersoncferreira/code-review/pull/85

**Reply by ag91** [reply]
thanks!
